.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "2.7"
+
+install:
+  - "gem install coveralls-lcov"
+  - "make deps"
+  - "pip install -e ."
+  - "pip install coveralls"
+
+script:
+  - "make test"
+
+after_success:
+  - "sed -i  's,SF:.,SF:ubcpi/static/js,' build/coverage/report-lcov/lcov.info"
+  - "coveralls-lcov -v -n build/coverage/report-lcov/lcov.info > js-coverage.json"
+  - "coveralls --merge=js-coverage.json"
```

Comment:
By default, travis runs on every commit to every branch, AND every commit to every pull request. This results in double builds on all pull requests, as you may have noticed. If you want to fix this, add these lines:

https://github.com/edx/i18n-tools/blob/master/.travis.yml#L17

to your configuration. That switches it to only run tests on commits to the master branch; builds will continue to run on every pull request as normal, so you won't be overloading travis infrastructure with needless double builds. (not sure why they made this the default :confused: )